src/cascadia/TerminalSettingsModel/JsonUtils.h

```diff
@@ -438,6 +438,72 @@ namespace Microsoft::Terminal::Settings::Model::JsonUtils
     };
 #endif
 
+    template<>
+    struct ConversionTrait<Json::Value>
+    {
+        Json::Value FromJson(const Json::Value& json)
+        {
+            return json;
+        }
+
+        bool CanConvert(const Json::Value&)
+        {
+            return true;
+        }
+
+        Json::Value ToJson(const Json::Value& val)
+        {
+            return val;
+        }
+
+        std::string TypeDescription() const
+        {
+            return "Json::Value";
+        }
+    };
+
+    template<typename T>
+    void SetValueForKey(Json::Value& json, std::string_view key, const T& target);
+
+    template<typename T>
+    std::decay_t<T> GetValue(const Json::Value& json);
+
+    template<>
+    struct ConversionTrait<winrt::Windows::Foundation::Collections::StringMap>

```

Comment:
If we move this `ConversionTrait` near the end of the file, would we need the fwddecls for nit: `SetValueForKey` and `GetValue` above?

Reply:
I believe this is a bit of a chicken and egg story. I can't put it below `SetValueForKey` or `GetValue` as they both use the `ConversionTrait` template. So while, yes, it could go lower I don't believe it can go low enough to not require the fwd declarations and then it wouldn't be near the other `ConversionTrait` template specialisations.

The other alternative is that I don't put this `ConversionTrait` specialisation in `JsonUtil.h`. It could go in `Profile.cpp` if you think this is better?